src/content/en/updates/2017/07/devtools-release-notes.md

```diff
@@ -0,0 +1,394 @@
+var feedback = {
+  "category": "DevTools",
+  "question": 'What do you think of this change?',
+  "choices": [
+    {
+      "button": {
+        "text": "I Like It"
+      },
+      "response": like,
+      "analytics": {
+        "label": label
+      }
+    },
+    {
+      "button": {
+        "text": "I Don't Like It"
+      },
+      "response": dislike,
+      "analytics": {
+        "label": label,
+        "value": 0
+      }
+    }
+  ]
+};
+</script>
+{% include "web/_shared/multichoice.html" %}
+{% endframebox %}
+
+## Set mousewheel behavior to zoom or scroll with Performance recordings {: #mousewheel }
+
+Open [Settings][Settings] and set the new **Flamechart mouse wheel action** setting to
+change how mousewheels behave on the **Performance** panel.
+
+[Settings]: /web/tools/chrome-devtools/ui#settings
+
+For example, when you use a mousewheel on the **Main** section of a recording,
+or when you swipe with two fingers on a trackpad, the default behavior is
+to zoom in or out. When you change the setting to **Scroll**, this gesture now
+scrolls up or down.
+
+<figure>
+  <img src="/web/updates/images/2017/07/flamechart-setting.svg"
+       alt="The 'Flamechart mouse wheel action' setting"/>
+  <figcaption>
+    <b>Figure 6</b>. The <b>Flamechart mouse wheel action</b> setting
+  </figcaption>
+</figure>
+
+{% framebox width="auto" height="auto" enable_widgets="true" %}
+<script>
+var like = 'Great. You\'re welcome to tweet us at ' +
+    '<a href="https://twitter.com/chromedevtools">@ChromeDevTools</a> or start a thread in the ' +
+    '<a href="https://groups.google.com/forum/#!forum/google-chrome-developer-tools">mailing ' +
+    'list</a> if you\'ve got any ideas on how to improve this.';
+var dislike = 'Sorry to hear that. Please tweet us at ' +
+    '<a href="https://twitter.com/chromedevtools">@ChromeDevTools</a> or start a thread in the ' +
+    '<a href="https://groups.google.com/forum/#!forum/google-chrome-developer-tools">mailing ' +
+    'list</a> and let us know what\'s wrong with it.';
+var label = 'WNDT61 / Mousewheel Behavior / Sentiment';
+var feedback = {
+  "category": "DevTools",
+  "question": 'What do you think of this change?',
+  "choices": [
+    {
+      "button": {
+        "text": "I Like It"
+      },
+      "response": like,
+      "analytics": {
+        "label": label
+      }
+    },
+    {
+      "button": {
+        "text": "I Don't Like It"
+      },
+      "response": dislike,
+      "analytics": {
+        "label": label,
+        "value": 0
+      }
+    }
+  ]
+};
+</script>
+{% include "web/_shared/multichoice.html" %}
+{% endframebox %}
+
+## Debugging support for ES6 Modules {: #modules }
+
+ES6 Modules are shipping natively in Chrome 61. There's not much going on here
+with regards to DevTools, other than that debugging works as you'd expect it
+to. Try setting some breakpoints in and stepping through [Paul Irish's
+ES6-Module-implementation of TodoMVC][TodoMVC] to see for yourself.
```

Comment:
Paul's code is a fork of something Addy (among others) did before he was at Google. http://todomvc.com/

Probably need to give proper credit.